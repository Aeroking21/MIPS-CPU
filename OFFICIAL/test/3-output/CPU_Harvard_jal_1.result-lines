RESULT =  00002222
