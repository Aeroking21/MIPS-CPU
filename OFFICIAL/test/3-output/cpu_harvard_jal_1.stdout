WARNING: test/ROM.v:23: $readmemh(test/1-binary/jal_1_instructions.mem): Not enough words in the file for the requested range [-1077936128:-1077933128].
WARNING: test/ROM.v:24: $readmemh(pre_reset_v.mem): Not enough words in the file for the requested range [-1077936136:-1077936128].
Instruction = 0f
Instruction r = 0f
ERROR: rtl/RAM.v:56: $readmemh: Unable to open test/5-data/data_jal_1.mem for reading.
VCD info: dumpfile CPU_TB.vcd opened for output.
ROM MODULE RESULTS:
CPU started
RESULT = 00001112
